src/posthog-surveys.ts

```diff
@@ -92,7 +90,8 @@ export class PostHogSurveys {
         }
 
         if (this._surveyEventReceiver == null) {
-            this._surveyEventReceiver = new SurveyEventReceiver(this.instance.persistence)
+            logger.error('SurveyEventReceiver is not initialized. Must init surveys before calling getSurveys')
+            return callback([])
```

Comment:
maybe we should throw here i guess its invalid to be in this state

or we move the events receiver back into the main bundle, it's only 1kb increase after all, not the main offender here